index.rst

```diff
@@ -2892,8 +2923,22 @@ average scan latencies as follows:
 
 - ``Object_Extras`` [#]_ queries (join): 8 hours.
 
-A dedicated queue (and consequently, in-memory space for corresponding
-chunks) will be managed for each scan.
+
+As stated in 8.10.2, there will schedulers for queries that are expected to
+take one hour, eight hours, or twelve hours and the schedulers group the
```

Comment:
I would make it a new sentence, too many "and"s in here. "The schedulers group [...]".